iroh-share/src/p2p_node.rs

```diff
@@ -173,10 +172,6 @@ impl P2pNode {
         let store_config = iroh_store::Config {
             path: db_path.to_path_buf(),
             rpc_client: rpc_store_client_config,
-            metrics: iroh_metrics::config::Config {
-                tracing: false, // disable tracing by default
```

Comment:
This is proof that the current state of things is weird.  This did nothing IIUC.